config/core/resources/sequence.yaml

```diff
@@ -0,0 +1,130 @@
+# Copyright 2019 The Knative Authors
```

Comment:
this is not needed anymore, but that's because I'm still working through the cleanup, I'll deal with it.